### PR TITLE
Improved drop stats docs

### PR DESCRIPTION
**Pull request by ethouris**

### CODE REVIEW

**Comment by mbakholdina** [comment]
@ethouris What about packets dropped at the sender side? I've left a comment with questions.

File: docs/statistics.md [hunk 1]
```diff
@@ -211,17 +211,17 @@ The total accumulated time in microseconds, during which the SRT sender has some
 
 The total number of "too late to send" packets dropped by the sender (refer to `SRTO_TLPKTDROP` in [API.md](API.md)). Available for sender.
 
+Packets may be dropped conditionally, when the `SRTO_TSBPDMODE` and `SRTO_TLPKTDROP` options are set to true.
+
 The total delay before TLPKTDROP mechanism is triggered consists of the `SRTO_PEERLATENCY`, plus `SRTO_SNDDROPDELAY`, plus 2 * the ACK interval (default ACK interval is 10 ms). The delay used is the timespan between the very first packet and the latest packet in the sender's buffer.
 
 ### pktRcvDropTotal
```
[the remark this answers is not in the document]
Comment (mbakholdina) [reply docs/statistics.md:218]: Should I then make a commit in your PR? Do I have rights?

File: docs/statistics.md [hunk 2]
```diff
@@ -211,17 +211,17 @@ The total accumulated time in microseconds, during which the SRT sender has some
 
 The total number of "too late to send" packets dropped by the sender (refer to `SRTO_TLPKTDROP` in [API.md](API.md)). Available for sender.
 
+Packets may be dropped conditionally, when the `SRTO_TSBPDMODE` and `SRTO_TLPKTDROP` options are set to true.
+
 The total delay before TLPKTDROP mechanism is triggered consists of the `SRTO_PEERLATENCY`, plus `SRTO_SNDDROPDELAY`, plus 2 * the ACK interval (default ACK interval is 10 ms). The delay used is the timespan between the very first packet and the latest packet in the sender's buffer.
```
[the remark this answers is not in the document]
Comment (ethouris) [reply docs/statistics.md:216]: The second sentence is kinda weird. The delay is calculated as shown (this results to more-less this 125% RFC is talking about), but this is simply about the time distance between now and the time recorded in the packet at the moment when it's first sent (as this time is a base to calculate the play time at the reception side).

File: docs/statistics.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ethouris) [reply docs/statistics.md:218]: Not sure, just try. If not, I'll make the change myself.

[the remark this answers is not in the document]
Comment (mbakholdina) [reply docs/statistics.md:218]: Done.

File: docs/statistics.md [hunk 3]
```diff
@@ -211,17 +211,19 @@ The total accumulated time in microseconds, during which the SRT sender has some
 
 The total number of "too late to send" packets dropped by the sender (refer to `SRTO_TLPKTDROP` in [API.md](API.md)). Available for sender.
 
+Packets may be dropped conditionally, when the `SRTO_TSBPDMODE` and `SRTO_TLPKTDROP` options are set to true.
```
Comment (maxsharabayko) [inline docs/statistics.md:214]:
```suggestion
Packets may be dropped conditionally when the `SRTO_TSBPDMODE` and `SRTO_TLPKTDROP` options are enabled.
```

**Comment by mbakholdina** [comment]
@maxsharabayko The PR is ready for merge.

**Pull request merged**